data/helpers.d/filesystem

```diff
@@ -62,15 +62,3 @@ ynh_bind_or_cp() {
     echo "This helper is deprecated, you should use ynh_backup instead" >&2
     ynh_backup "$1" "$2" 1 "$NO_ROOT"
 }
-
-# Create a directory under /tmp
-#
-# usage: ynh_mkdir_tmp
-# | ret: the created directory path
-ynh_mkdir_tmp() {
-    TMPDIR="/tmp/$(ynh_string_random 6)"
-    while [ -d $TMPDIR ]; do
-        TMPDIR="/tmp/$(ynh_string_random 6)"
-    done
-    mkdir -p "$TMPDIR" && echo "$TMPDIR"
```

Comment:
race condition possible between this line and the previous one.
if another process creates the same dir between the two lines, boom.